### ISSUE TITLE
Oasis doesn't remember my preferred color scheme.

### ISSUE THREAD

**Issue by aadilayub** [fraction/oasis#333]
## What's the problem you want solved?
I've noticed that Oasis switches back to the default color scheme every time I launch it (I usually like setting it to `monokai`). It would be cool if my choice of color scheme persists between sessions.

## Is there a solution you'd like to recommend?
Maybe whenever someone sets their preferences through the settings menu in oasis, these settings get saved in the `default.json` file.

## What version or commit of Oasis are you using?
2.14.9

**Comment by aadilayub** [comment]
I just tried closing and launching oasis. This time the color scheme choice persisted. Maybe this is an issue that only happens between reboots.....

**Comment by christianbundy** [comment]
Yep, these are being set as cookies but we could persist them elsewhere instead.

**Comment by georgeowell** [comment]
> Yep, these are being set as cookies but we could persist them elsewhere instead.

I also have the issues @aadilayub has with cookies.

We could store in the config maybe? It seems like something that doesn't change too often.

**Comment by christianbundy** [comment]
Oops, I shouldn't have closed this. I think the next step is to start writing the theme preference to the config file.
